pkgs/games/unciv/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchurl
+, makeDesktopItem
+, makeWrapper
+, jre
+, libpulseaudio
+, libXxf86vm
+}:
+let
+  desktopItem = makeDesktopItem {
+    name = "unciv";
+    exec = "unciv";
+    comment = "An open-source Android/Desktop remake of Civ V";
+    desktopName = "Unciv";
+    categories = "Game;";
+  };
+
+  envLibPath = stdenv.lib.makeLibraryPath [
+    libpulseaudio
+    libXxf86vm
+  ];
+
+in
+stdenv.mkDerivation rec {
+  pname = "unciv";
+
+  version = "3.11.8";
+
+  src = fetchurl {
+    url = "https://github.com/yairm210/Unciv/releases/download/${version}/Unciv.jar";
+    sha256 = "14l54nj075rmy34li7gfr7b5v6wpvb81i4h2xlv9jj2ps38rk4yy";
+  };
+
+  phases = "installPhase";
+
```

Comment:
```suggestion

```

Reply:
Removed it according to your suggestion and got this:
```
building '/nix/store/rynrlpyag3zj0gqpnbsr8gvlky3s5887-unciv-3.12.8.drv'...
unpacking sources
unpacking source archive /nix/store/xkimqn9978bpfy9bf0cnx0bk7blcygk1-Unciv.jar
do not know how to unpack source archive /nix/store/xkimqn9978bpfy9bf0cnx0bk7blcygk1-Unciv.jar
builder for '/nix/store/rynrlpyag3zj0gqpnbsr8gvlky3s5887-unciv-3.12.8.drv' failed with exit code 1
```